.storybook/webpack.config.js

```diff
@@ -1,23 +1,10 @@
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin')
+
 module.exports = ({ config }) => {
-  // ts-config.
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    use: [
-      {
-        loader: require.resolve('awesome-typescript-loader'),
```

Comment:
Do we not still need a ts loader for webpack?

Reply:
No, the new storybook supports typescript out of the box